docs/conf.py

```diff
@@ -252,8 +252,9 @@
 # NOTE: By default, the following MyST extensions are enabled:
 #       substitution, deflist, linkify
 
-# myst_enable_extensions = set()
-
+myst_enable_extensions = [
+    "colon_fence"
+]
```

Comment:
```suggestion
myst_enable_extensions = {
    "colon_fence"
}
```

Needs to be this way due to an issue with the canonical-sphinx extension and how it combines these with the defaults.